raw_data_QA/check_raw_data.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
```

Comment:
I'd re-name this file to be more specific as it's checking nytimes data only.

Reply:
Good point, I removed this and renamed it in the last commit